.gitignore

```diff
@@ -52,3 +52,4 @@ java/.metadata/.plugins/org.eclipse.jdt.core/assumedExternalFilesCache
 java/.metadata/.plugins/org.eclipse.jdt.core/externalFilesCache
 java/.metadata/.plugins/org.eclipse.jdt.core/nonChainingJarsCache
 java/.metadata/.plugins/org.eclipse.ui.intro/introstate
```

Comment:
Please remove all those lines and just put `java/*` inside the gitignore

Reply:
correct as requested. (These come from the pull from another developer. I should review before pulling. thanks)